src/Admin/Tools/FeatureFlag/FeatureFlagManager.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Admin\Tools\FeatureFlag;
+
+use App\DB\Entity\FeatureFlag;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class FeatureFlagManager
+{
+  private array $defaultFlags;
+
+  public function __construct(
+    protected RequestStack $requestStack,
+    protected EntityManagerInterface $entityManager
+  ) {
+    $this->defaultFlags = include __DIR__.'/../../../../config/features.php';
```

Comment:
symfony has the parameter bag interface, which allows you to get variables such as the root directory; this would make the path more readable than stacking ../